### PR TITLE
test: add benchmark for model

### PR DESCRIPTION
**Pull request by Shivansh-yadav13**
- Demo PR (WIP)
- added benchmarks for model
- test/benchmark/model.ts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
   "peerDependencies": {},
   "dependencies": {
     "await-lock": "^2.0.1",
+    "benny": "^3.7.1",
```
Comment (nodece) [inline package.json]: Move this to `devDependencies`.

**Review by nodece — CHANGES_REQUESTED** [review]
Good work, could you add a workflow for the benchmark?

**Comment by Shivansh-yadav13** [comment]
> Good work, could you add a workflow for the benchmark?

@nodece pls review, it will save a result file for each suite. eg: `test/benchmark/results/RBACModel.json`

**Comment by nodece** [comment]
By the way, you should use `test` instead of `feat`, so like `test: add benchmark for model`.

**Review by nodece — APPROVED** [review]

**Comment by nodece** [comment]
Please check CI.

**Comment by nodece** [comment]
It looks like you updated other dependencies in yarn.lock file, please rollback the package.json and yarn.lock, then add benny by `yarn add benny -D` command.

**Comment by Shivansh-yadav13** [comment]
I have fixed the CI

**Review by nodece — APPROVED** [review]

**Pull request merged**